react/Dropdown/Dropdown.js

```diff
@@ -116,19 +128,51 @@ export default class Dropdown extends Component {
     const { id, className, valid, tone } = this.props;
     const classNames = classnames({
       [styles.root]: true,
-      [styles.invalid]: typeof tone !== 'undefined' ? tone === TONE.CRITICAL : valid === false,
+      [styles.invalid]:
+        typeof tone !== 'undefined' ? tone === TONE.CRITICAL : valid === false,
       [className]: className
     });
 
-    // eslint-disable-next-line react/prop-types
-    const { label, labelProps, secondaryLabel, tertiaryLabel, invalid, help, helpProps, message, messageProps } = this.props;
+    const {
+      // eslint-disable-next-line react/prop-types
```

Comment:
Might be cleaner to wrap this in disable/enable block comments

Reply:
Happy to. Was avoiding over-doing changes, but this seems fine.